src/mutation/mutation.js

```diff
@@ -83,6 +85,7 @@ export function mutationWithClientMutationId(
 
   return {
     type: outputType,
+    description: description,
```

Comment:
This field is optional in `GraphQLFieldConfig`, so we should probably omit it rather than return an empty string.